### PR TITLE
migrate to using tracing instead of log

### PR DESCRIPTION
**Pull request by syphar**
in preparation for the async web framework migration ( and sqlx) this migrates our logging to tracing. 

Currently not changing any log-message, the key-value and `#[instrument]` awesomeness can be added step by step in later PRs.

### CODE REVIEW

File: src/docbuilder/rustwide_builder.rs [hunk 1]
```diff
@@ -160,8 +160,8 @@ impl RustwideBuilder {
         // NOTE: this ignores the error so that you can still run a build without rustfmt.
         // This should only happen if you run a build for the first time when rustfmt isn't available.
         if let Err(err) = self.toolchain.add_component(&self.workspace, "rustfmt") {
-            log::warn!("failed to install rustfmt: {}", err);
-            log::info!("continuing anyway, since this must be the first build");
+            tracing::warn!("failed to install rustfmt: {}", err);
+            tracing::info!("continuing anyway, since this must be the first build");
```
Comment (jyn514) [inline src/docbuilder/rustwide_builder.rs:164]: can we import these macros to avoid churn in the future?

**Review by jyn514 — APPROVED** [review]

File: src/docbuilder/rustwide_builder.rs (hunk 1 above)
Comment (syphar) [reply src/docbuilder/rustwide_builder.rs:164]: will do so

**Comment by syphar** [comment]
@jyn514 I added a commit moving to imports, fine with you as I did it?

**Comment by jyn514** [comment]
looks great, thanks!

**Pull request merged**